src/pages/orgs/index.tsx

```diff
@@ -144,15 +144,14 @@ const OrgRouter = () => {
                         sx={{
                             width: "100%",
                             marginTop: "20px",
-                            paddingLeft: "20px",
                         }}
                     >
                         <AsyncButton
                             onClick={() => navigate("/catalog")}
                             variant="contained"
-                            sx={{ width: "80px" }}
+                            sx={{ width: "200px" }}
                         >
-                            Back
+                            Back to Catalogue

```

Comment:
For consistency with everywhere else (including URL slugs) we should probably use the American spelling "catalog"